renderdoc/android/android.cpp

```diff
@@ -487,31 +528,34 @@ RDResult InstallRenderDocServer(const rdcstr &deviceID)
 
     RDCLOG("Installed package '%s', checking for success...", apk.c_str());
 
-    AndroidVersionCheckResult check = CheckAndroidServerVersion(deviceID, abi);
+    AndroidVersionCheckResult versionCheck = CheckAndroidServerVersion(deviceID, abi);
+    AndroidInstallPermissionCheckResult permissionsCheck =
```

Comment:
Is it possible to only do the `permissionsCheck` if `versionCheck == Correct` ?
Trying to reduce the amount `adb` work to the minimum.
My logic is if the version check has failed then don't need to do the permissionsCheck as the install will be re-triggered

Reply:
This is reasonable since we are dealing with an edge case. I have hopefully made this logic a bit more explicit around this.